python/python/glide/async_commands/cluster_commands.py

```diff
@@ -367,3 +368,87 @@ async def lastsave(self, route: Optional[Route] = None) -> TClusterResponse[int]
             TClusterResponse[int],
             await self._execute_command(RequestType.LastSave, [], route),
         )
+
+    async def sort(
```

Comment:
Why is it in ClusterCommands and not in CoreCommands?

Reply:
Because we have 2 function args which aren't supported in cluster mode and we decided to not let the user send them.
For this we had to split the impl to Cluster and Standalone modes.